Doc/library/datetime.rst

```diff
@@ -804,6 +811,11 @@ Other constructors, all class methods:
    except the latter formula always supports the full years range: between
    :const:`MINYEAR` and :const:`MAXYEAR` inclusive.
 
+   .. warning::
+
+      Because naive datetimes are treated by many functions as local times.
+      To get an aware :class:`.datetime` object, use :meth:`fromtimestamp`.
```

Comment:
The Sphinx role here for ``datetime`` can be removed here since this section is already within the ``datetime`` class. The main purpose of the roles is to provide links to other sections, so there's not much of a purpose in a role that links to a section it is already contained within.

```suggestion
      To get a timezone specific datetime object, use :meth:`fromtimestamp`.
```